mbs-s0015-20240504/check_spectra_mpi.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import numpy as np
+from mpi4py import MPI
+from pixell import enmap, curvedsky, mpi as p_mpi
+from mnms import utils
+
+opj = os.path.join
+comm = MPI.COMM_WORLD
+
+imapdir = '/mnt/home/aduivenvoorden/project/so/20240504_mss0002_lat/renamed'
```

Comment:
@AdriJD sorry, noticed just now, can you use the `cfdirs` path instead of your home folder?

Reply:
I added the paths on NERSC as comments, and added clarification that the original paths in the scripts were paths on the CCA cluster. I didn't generate the simulations on NERSC, so I wanted to keep the original paths in the scripts as well as a reference. I hope that is a good compromise.